### DISCUSSION TITLE
HTTP 地址下的语音输入没有反应

### DISCUSSION THREAD

**Discussion by whileiwin** [lobehub/lobehub#3249] [General Question]
### 📦 部署环境

Docker

### 📌 软件版本

v1.2.10

### 💻 系统环境

Windows

### 🌐 浏览器

Edge

### 🐛 问题描述

为什么自己在DOCKER布置的lobechat在对话框中点语音输入时一直显示“润色中”，语音输入没有反应。已经填入了可用的openaikey。 测试了https://chat-preview.lobehub.com/官方布置的可以正常语音，说明不是MIC的问题，两者设置上我也没有看出明显的区别，是还有什么地方要设置的吗？

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
这个没遇到过哎… 看下是不是 https 协议的问题？

[a comment by an automated account is not shown]

**Comment by one-pyy** [comment]
可能是因为浏览器在http的时候不允许你录音（哪怕你给权限也不行）

参考https://blog.csdn.net/first_shun/article/details/106343825

[a comment by an automated account is not shown]

**Comment by JoeChen2me** [comment]
同样，我也有这个问题，版本v1.4.3。无论是数据库部署还是客户端部署都有这个问题，语音输入显示润色中...。
